File: assets/commands.py
from botty import BotCommand

from api import Command


class Groups:
    FOR_USERS = [
        BotCommand(Command.START, "Главное меню"),
        BotCommand(Command.GET_AVATARS, "Получить аватарку"),
        BotCommand(Command.GET_PAIRED, "Получить парные аватарки"),
        BotCommand(Command.GET_CUTE, "Получить милую пикчу"),
        BotCommand(Command.GET_ANGRY, "Получить агрессивную пикчу"),
        BotCommand(Command.SEND_PICTURE, "Отправить пикчу пользователю"),
    ]
    FOR_ADMINS = FOR_USERS + [
        BotCommand(Command.ADMIN, "Админ-панель"),
    ]

File: assets/commands.ts
import { BotCommand } from "tg"
import { Command } from "enums"

function C(cmd: Command, text: string): BotCommand {
  return { command: cmd, description: text }
}

class Groups {
  FOR_USERS = [
    C("start", "Главное меню"),
    C("get_avatars", "Получить аватарку"),
    C("get_paired", "Получить парные аватарки"),
    C("get_cute", "Получить милую пикчу"),
    C("get_angry", "Получить агрессивную пикчу"),
    C("send_picture", "Отправить пикчу пользователю"),
  ]
  FOR_ADMINS = this.FOR_USERS.concat([
    C("admin", "Админ-панель"),
  ])
}

export const groups = new Groups()
